src/openstack_cloud/openstack_manager.py

```diff
@@ -487,6 +494,11 @@ def _health_check(conn: OpenstackConnection, server_name: str, startup: bool = F
             return False
         if server.status not in (_INSTANCE_STATUS_ACTIVE, _INSTANCE_STATUS_BUILDING):
             return False
+        created_at = datetime.strptime(server.created_at, "%Y-%m-%dT%H:%M:%SZ")
+        current_time = datetime.now(created_at.tzinfo)
+        elapsed_min = (created_at - current_time).total_seconds() / 60
```

Comment:
What is the current timeout?

Reply:
5 minutes is the current timeout. Good catch, there already is a constant for it.